egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -37,16 +52,21 @@ if [ $stage -le 1 ]; then
     else
       num_reps=1
     fi
-    reverb_data_dirs=
-    for i in `seq 1 $num_reps`; do
-      cur_dest_dir=" data/temp_${data_dir}_${i}"
-      local/multi_condition/reverberate_data_dir.sh --random-seed $i \
-        --snrs "$snrs" --log-dir exp/make_corrupted_wav \
-        data/${data_dir}  data/impulses_noises $cur_dest_dir
-      reverb_data_dirs+=" $cur_dest_dir"
-    done
-    utils/combine_data.sh --extra-files utt2uniq data/${data_dir}_rvb $reverb_data_dirs
-    rm -rf $reverb_data_dirs
+
+    python steps/data/reverberate_data_dir.py \
+      --prefix "rev" \
+      --rir-set-parameters "1.0, SIM_RIR/simulated_rir_mediumroom/rir_list" \
```

Comment:
Also in the interest of making scripts easy to copy, keep everything related to data augmentation in a sub-directory like `local/multi_condition/reverberate/`. Most of the scripts in the current multi_condition directory can be moved to this script.

@danpovey People trying out data reverberation in other recipes like Babel, were copying the rir, noise prep scripts from ASpIRE. This replication is not desirable, so can we find a home for these scripts in the `steps/` directory ?